main.py

```diff
@@ -67,6 +71,8 @@ def __init__(self):
         super().__init__()
         self.withdraw()
 
+        self.check_for_updates()
```

Comment:
Put this after the deiconify. Making a messagebox whithout a viewable window causes issues on other systems.